portfolio/src/main/webapp/script.js

```diff
@@ -25,31 +27,38 @@ $(document).ready(function () {
 });
 
 const animateScrollTop = (targetClass) => {
-  $("html").animate(
-    {
+  $("html").animate({
       scrollTop: $("." + targetClass).offset().top,
     },
     2000
   );
 };
 
+async function getComments() {
+  const response = await fetch("/data", {
+    method: "GET"
+  });
+  const jsonResponse = await response.text();
+  const jsonComments = JSON.parse(jsonResponse);
+  const comments = jsonComments["comments"];
+  for (let index = 0; index < comments.length; index++) {
+    const comment = comments[index];
+    $('#comments-container').append(`<p>${comment.comment}</p>`);
+    $('#comments-container').append(`<p>${comment.name}</p>`);
```

Comment:
The same code repeated here and below. Consider putting this in its own function.

Reply:
done